testify/test_program.py

```diff
@@ -61,8 +61,9 @@ def load_plugins():
                 # Need some unlikely-to-clash unique-ish module name
                 mod_name = '_testify_plugin__' + mod_name
 
-                try:
-                    SourceFileLoader(mod_name, full_file_path).load_module()
+                try:                    
+                    module_type = SourceFileLoader(mod_name, full_file_path).load_module()
+                    plugin_modules.append(module_type)
```

Comment:
@StefanoChiodino Could this be a one-liner like this: 
`plugin_modules.append(SourceFileLoader(mod_name, full_file_path).load_module())`

Reply:
if you try hard enough, you can fit the whole file in a single line 😂

Reply:
And then you're writing perl ;)